src/auth/ducks/thunks.ts

```diff
@@ -38,6 +40,7 @@ export const signup = (
         AppAxiosInstance.defaults.headers['X-Access-Token'] =
           response.accessToken;
         dispatch(authenticateUser.loaded(response));
+        authClient.setContacts(contactInfo, response.accessToken);
```

Comment:
interesting, this works but separating them out solves the idempotency issue, which we can't otherwise account for here.